.travis.yml

```diff
@@ -2,12 +2,22 @@ language: python
 python:
 - '3.6'
 cache:
-- pip
+  directories:
+  - $HOME/.cache/pip
+  - .arca/cache
 install:
 - pip install -r test_requirements.txt
+# https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions
+# before_script:
+# - docker login -u "$DOCKER_HUB_USERNAME" -p "$DOCKER_HUB_PASSWORD"
 script:
 - python -m pytest test_naucse
+- python -m naucse list_courses
```

Comment:
Please add a comment on why you're adding this – the pull request description isn't kept with the code